doc/releases/release-notes-2.1.rst

```diff
@@ -141,4 +660,305 @@ release:
+* :github:`16320` - The routing option CONFIG_NET_ROUTING needs clarification
+* :github:`16223` - stm32: Unable to send 64 byte packet over control endpoint
+* :github:`16167` - Implement interrupt driven GPIO on LPC families
+* :github:`16097` - STM32 Ethernet driver should be able to detect the carrier state
+* :github:`16032` - Socket UDP: Low transmission efficiency
+* :github:`16031` - Toolchain abstraction
+* :github:`15912` - add Reject as an option to pull request reviews
+* :github:`15906` - WEST ERROR: extension command build was improperly defined
+* :github:`15841` - Support AT86RF233
+* :github:`15604` - Suspicious PCI and build_on_all default test coverage
+* :github:`15603` - Unable to use C++ Standard Library
+* :github:`15598` - Standard devicetree connectors for boards
+* :github:`15551` - CMake enables -fmacro-prefix-map on GCC 7
+* :github:`15494` - 2.0 Release Checklist
+* :github:`15359` - The docs incorrectly state that common.dts integrates with mcuboot
+* :github:`15323` - blink_led sample does not work on most of the nRF boards
+* :github:`15246` - doc: confusion about dtc version
+* :github:`15196` - logging: Support for blocking deferred logging
+* :github:`15027` - doc: PDF generation broken
+* :github:`14906` - USB: NXP Device controller does not pass testusb tests
+* :github:`14810` - DT_<Driver Name>_LABEL undeclared
+* :github:`14744` - ARM: Core Stack Improvements/Bug fixes for 2.0 release
+* :github:`14683` - need end-to-end memory protection samples
+* :github:`13725` - drivers: ssd1306: When 128x32 is used, only half of the screen is output.
+* :github:`13000` - sanitycheck serializes running tests on ARC simulator
+* :github:`12969` - settings: loading key-value pairs for given subtree
+* :github:`12965` - POSIX subsys: Need more fine-grained enable options
+* :github:`12961` - ARM Memory Protection functions not invoked in SWAP for ARMv6/ARMv8-M Baseline
+* :github:`12933` - MCUboot: high current
+* :github:`12813` - DTS: CONFIG_FLASH_BASE_ADDRESS not being generated for SPI based Flash chip
+* :github:`12703` - how to configure interrupt signals on shields via device tree?
+* :github:`12677` - USB: There are some limitations for users to process descriptors
+* :github:`12535` - Bluetooth: suspend private address (RPA) rotating
+* :github:`12509` - Fix rounding in _ms_to_ticks()
+* :github:`12504` - STM32: add USB_OTG_HS example
+* :github:`12206` - OpenThread apps want to download and build OpenThread every time!
+* :github:`12114` - assertion using nRF5 power clock with BLE and nRF5 temp sensor
+* :github:`11743` - logging: add user mode access
+* :github:`11717` - qemu_x86 's SeaBIOS clears the screen every time it runs
+* :github:`11655` - Alleged multiple design and implementation issues with logging
+* :github:`10748` - Work waiting on pollable objects
+* :github:`10701` - API: Prefix (aio_) conflict between POSIX AsyncIO and Designware AnalogIO Comparator
```

Comment:
The script that creates these lists of issues needs to scan for meaningful reST syntax and escape those so they're not interpreted by Sphinx.  In this case replace ``_`` with ``\_``

Reply:
ok